opentelemetry-otlp/examples/basic-otlp-http/docker-compose.yaml

```diff
@@ -21,17 +13,7 @@ services:
       - "4317:4317"   # OTLP gRPC receiver
```

Comment:
we don't need gRPC as this is only for http.. maybe better to combine both to a single example, as switching from http to gRPC is a small change in exporter.

Reply:
Yes, this PR is meant to add the log exporter in existing OTLP HTTP example. We can consolidate both the examples into single as separate cleanup PR.

Reply:
Have removed the unwanted ports from both the examples now.